content/en/cloud_cost_management/recommendations/_index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Cloud Cost Recommendations
+kind: documentation
+private: true
+description: Learn how to reduce the spending of your organization's cloud resources with Cost Recommendations.
+further_reading:
+- link: "/cloud_cost_management/"
+  tag: "Documentation"
+  text: "Learn about Cloud Cost Management"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Cloud Cost Management is not supported for this site.</div>
+{{< /site-region >}}
+
+{{< callout url="#" btn_hidden="true" >}}
+Cloud Cost Recommendations is in public beta with support for AWS, and is automatically enabled if you have set up <a href="/cloud_cost_management/">Cloud Cost Management</a>.
+{{< /callout >}}
+
+## Overview
+
+[Cloud Cost Recommendations][1] provides recommendations on reducing your cloud spending by optimizing the usage of your cloud resources. 
+
+{{< img src="cloud_cost/recommendations/cost_recommendations.png" alt="Overview tab with potential monthly savings, potential annual savings, and total number of open cases on the Cloud Cost Recommendations page" style="width:100%;" >}}
+
+Recommendations combines billing data and observability data to identify orphaned, legacy, or over-provisioned cloud resources.
+
+## Setup
+
+For each AWS account that you would like to receive recommendations for:
+
+1. Configure [Cloud Cost Management][2] to start sending bill data to Datadog.
+1. Enable [resource collection][3] in the **Resource Collection** tab on the [AWS integration tile][4].
+1. Install the [Datadog Agent][5] (required for over-provisioned resource recommendations).
+
+## Recommendation types
+
+Datadog generates a set of recommendations by combining your observability data with your underlying cloud provider's billing data. You can see the detailed logic for each recommendation type, along with observability metrics or cost data used to generate the recommendation [on the **Recommendations** page][1].
+
+{{< img src="cloud_cost/recommendations/resource_side_panel.png" alt="Cloud cost allocation table showing requests and idle costs over the past month on the Containers page" style="width:100%;" >}}
+
+Recommendations are run on a daily basis, and are automatically refreshed in your account. When new recommendations are released, Datadog automatically adds them to your account. 
+
+### Terminate resource recommendations
+
+Datadog scans your cloud environment to identify and surface orphaned resources that can be removed. 
+
+Low Utilization EC2 Instances
+: EC2 instances with less than 5% CPU utilization, and less than 10% memory utilization. 
+
+Unattached EBS Volumes Unattached
```

Comment:
```suggestion
Unattached EBS Volumes
```
Or `EBS Volumes Unattached`?

Reply:
Confirming that you caught this correctly in the UI!

![Screenshot 2024-04-09 at 11 17 51 AM](https://github.com/DataDog/documentation/assets/76412946/5320763c-8f6d-41be-bc17-ab4638ef80fd)